tosan-httpserver-spring-boot-starter/src/main/java/com/tosan/http/server/starter/filter/HttpStatisticsFilter.java

```diff
@@ -81,9 +84,14 @@ private void end(String serviceName, long startTime) {
         double duration = (System.currentTimeMillis() - startTime) / 1000.0;
         final Map<String, Object> outwardLog = new LinkedHashMap<>();
         outwardLog.put("-service", serviceName);
-        outwardLog.put("duration", duration + "s");
+        List<ServiceExecutionInfo> serviceExecutionInfos = Statistics.getCurrentStatistics().getServiceExecutionsInfo();
+        if (!serviceExecutionInfos.isEmpty()) {
+            outwardLog.put("internal service call duration", serviceExecutionInfos);
```

Comment:
maybe at first more important data is better to be logged like total duration and active requests. log volume must be reduced as possible. so instead of internal service call duration we can go with statistics.
in logging service excecution info maybe we can go with serviceType.serviceName = duration....
consider that service type might be null.

cm sample of logging statistics:
2022-10-04T13:00:00.115 [INFO]  requestId:[NW-3mqkqJOo] channel:[10.5.35.123    ] thread:[qtp523794808-445527      ]  c.k.e.c.exporter.ServiceExporterBase.a@237:
    { "statistics" : "ePayment.ThirdPartyGetPaymentServices", "TrackerAspect" : "0.023s", "LoadSessionAspect" : "0.0s" } 
2022-10-04T13:00:00.115 [INFO]  requestId:[NW-3mqkqJOo] channel:[10.5.35.123    ] thread:[qtp523794808-445527      ]  c.k.e.c.exporter.ServiceExporterBase.a@212:
    { "-service" : "ePayment.ThirdPartyGetPaymentServices", "duration" : "0.066s", "operational active requests" : 2, "report active requests" : 0 }